### PR TITLE
[doc] Document roadmap

### PR DESCRIPTION
**Pull request by JingsongLi**
image

### CODE REVIEW

File: docs/content/project/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Roadmap
+
+Paimon's long-term goal is to become the better data lake platform for building the Streaming Lakehouse. Paimon will
+invest in real-time, ecology, and data warehouse integrity for a long time.
+
+If you have other requirements, please contact us.
+
+## What’s Next?
+
+### Core
```
Comment (SteNicholas) [inline docs/content/project/roadmap.md:36]: Could the metrics of Paimon add into the core plan?

File: docs/content/project/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Integration with Apache Seatunnel
+
+### Compute Engines
+- Flink DELETE/UPDATE support
+- More management DMLs to Flink/Spark `CALL` procedures. 
+- Hive Writer
+- Spark Writer supports `INSERT OVERWRITE`
+- Spark Time Traveling
+- Presto Reader
+- Apache Doris Reader
```
Comment (SteNicholas) [inline docs/content/project/roadmap.md:55]:
```suggestion
- Doris Reader
```

File: docs/content/project/roadmap.md (hunk 1 above)
Comment (JingsongLi) [reply docs/content/project/roadmap.md:36]: Recently I thought of a metrics, such as the busyness of compaction threads, which could allow users to notice and scale up in advance.

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**